### ISSUE TITLE
Pokemon Soul Silver Freezing

### ISSUE THREAD

**Issue by sleepyxd** [ahezard/nds-bootstrap#736]
### Your console

Tell us which console you are using:

- [ ]  DS Phat/Lite
- [ ]  DSi (XL)
- [x]  Old 3DS/2DS (XL)
- [ ]  New 3DS/2DS (XL)

### CFW and configuration

Tell us which CFW do you have:

DSi:

- [ ]  (No CFW) DSiWarehax (such as Flipnote Lenny, Sudokuhax, etc.)
- [ ]  (No CFW) Unlaunch only _[please specify version]_
- [ ]  Unlaunch+HiyaCFW _[please specify versions of both]_

3DS/2DS:

- [x]  Luma3DS
- [ ]  Rei-Six
- [ ]  Other/Older CFW [specify]

### nds-bootstrap version

Tell us which versions you are using:

- [x]  Release [v0.23.0]
- [ ]  Nightly _[insert here commit hash]_

### Your device

Tell us which card you are using:

- [x]  SD Card
- [ ]  (Slot-1) Flashcard

### If this issue only affects a certain game, which game is it?

Pokemon - Soul Silver Version

### Expected behaviour

Game should be playable after saving/Map Transition

### Actual behaviour

After saving: sometimes the game is playable as normal but more often than not the touch screen with turn black when it should transition back to the normal menu screen. Forcing a restart as no button inputs do anything.

Map Transitions: Around every 3-4 times walking through doors between switching map locations the screen will remain black instead of displaying the next map. Music still plays as normal but a restart is needed as nothing works.

### Steps to reproduce

1. Run Soul Silver through Twilight Menu (NTR/TWL Clock speeds & VRAM Boost doesn't seem to matter)
2. Game should appear to run as normal
3. Change map locations (walk through doors 2-3 times) or Save Game after switching maps once or twice.

### Other notes

N/A

**Comment by RocketRobz** [comment]
What TWLMenu++ version are you using?

**Comment by sleepyxd** [comment]
Latest

**Comment by RocketRobz** [comment]
Latest being?

**Comment by sleepyxd** [comment]
9.1.2

**Comment by RocketRobz** [comment]
Weird. I was thinking it's an AP-issue, but since TWLMenu++ already includes an AP-patch for Soul Silver, there should be no freezing issues.
Is the ROM modified in any way?

**Comment by sleepyxd** [comment]
Yeah, I used a hex editor to uncap the framerate. Although I was still having this issue without it.
